terraform/modules/services/s3-virus-scan/ecs.tf

```diff
@@ -0,0 +1,48 @@
+module "globals" {
+  source      = "../../globals"
+  environment = var.environment
+}
+
+data "template_file" "app_s3_virus_scan" {
+  template = file("${path.module}/s3_virus_scan.json.tpl")
+
+  vars = {
+    app_image                  = "${module.globals.env_accounts["mgmt"]}.dkr.ecr.eu-west-2.amazonaws.com/scale/s3-virus-scan:${var.ecr_image_id_spree}"
```

Comment:
Should use new var ` ecr_image_id_s3_virus_scan` for the image?

Reply:
@tommyb82  yes, this should now be updated to use the new var `ecr_image_id_s3_virus_scan` for the image